### DISCUSSION TITLE
Can any such API be created so that we can embed in our website and tell people where and how many times malware attacks have happened in the whole universe today?

### DISCUSSION THREAD

**Discussion by cybersecuritynews** [community/community#50216] [Code Security]
### Select Topic Area

Question

### Body

I have a brand cybersecurity blog called The Cyber Express, we put articles about cyber security news attacks and describe people, so I want some feature that I can put in my cyber security news blog. I am leaving the link of my website here, if anyone can tell and do it, please tell [removed by moderator]

**Comment by tuves** [comment]
Hi @cybersecuritynews, we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
